core/lib/basic_types/src/protocol_version.rs

```diff
@@ -101,7 +103,7 @@ impl ProtocolVersionId {
 
     pub fn is_pre_shared_bridge(&self) -> bool {
         // TODO: review this when we actually deploy shared bridge
-        true
+        false
```

Comment:
Leaving this comment to not forget to update it, since not every protocol version is not pre shared bridge